### PR TITLE
Update Agile methodologies - Fix the incorrect responses to Qs 8 & 9

### PR DESCRIPTION
**Pull request by rradfar**
### Question 8 explanation: 

Weighted Shortest Job First (WSJF) is a prioritization model used to **sequence jobs** (eg., Features, Capabilities, and Epics) to produce the **maximum economic benefit**. In a flow-based system, priorities are updated continuously to provide the best economic outcomes. Job sequencing, **rather than individual job return on investment**, produces the best result. To that end, WSJF is used to prioritize backlogs by calculating the relative CoD and job size (a proxy for the duration).

Source: [https://www.scaledagileframework.com/wsjf/](https://www.scaledagileframework.com/wsjf/)

### Question 9 explanation: 

It is suggested that **high business value, high-risk items are worked on first**. While that may seem counterintuitive, the earlier this work is done, the sooner the team will move to mitigate the issues and unknowns—leading to a higher quality product. If there's a failure, it will occur early and relatively inexpensively.

Source: [https://agilescrumguide.com/blog/files/tag-product-backlog---business-value-and-risk.html](https://agilescrumguide.com/blog/files/tag-product-backlog---business-value-and-risk.html)

### CODE REVIEW

File: agile-methodologies/agile-methodologies-quiz.md [hunk 1]
```diff
@@ -43,15 +43,15 @@
 - [ ] return on investment
 
 #### Q8. Why should you apply Weighted Shortest Job First?
-- [x] to maximize the return on investment
-- [ ] to determine the economic sequencing of the backlog
+- [ ] to maximize the return on investment
+- [x] to determine the economic sequencing of the backlog
```
Comment (Ebazhanov) [inline agile-methodologies/agile-methodologies-quiz.md:47]: @rradfar can you please transfer your explanation here with a reference link

File: agile-methodologies/agile-methodologies-quiz.md [hunk 2]
```diff
@@ -43,15 +43,15 @@
...
 - [ ] to visualize queue length
 - [ ] to fulfill a commitment to quality
 
 #### Q9. You have classified the features in your backlog according to risk and value. Which features should the development team work on first?
 - [ ] low-value and high-risk
-- [ ] high-value and high-risk
-- [x] high-value and low-risk
+- [x] high-value and high-risk
```
Comment (Ebazhanov) [inline agile-methodologies/agile-methodologies-quiz.md:53]: @rradfar the same here would be great to share your ideas with link

**Review by Ebazhanov — CHANGES_REQUESTED** [review]
@rradfar I left some comments but the general idea is to move your explanation into the *.md file. Thank you in advance!

**Comment by rradfar** [comment]
I made the changes you requested. Then I did a rebase in my local repo and that might've inadvertently messed something up because I see that this pull request now shows 11 files were changed (it shows other people's changes as well as mine) instead of 1 (which is the agile-methodologies-quiz.md file). Please let me know if this is okay, or if you'd like me to close this PR and create a new one. I'm still learning the process :) Cheers.

**Comment by Ebazhanov** [comment]
> I made the changes you requested. Then I did a rebase in my local repo and that might've inadvertently messed something up because I see that this pull request now shows 11 files were changed (it shows other people's changes as well as mine) instead of 1 (which is the agile-methodologies-quiz.md file). Please let me know if this is okay, or if you'd like me to close this PR and create a new one. I'm still learning the process :) Cheers.

Yes, let's close it and create a new one, please. Thank you! No problem you can train yourself here, this place is special for it!!!

**Pull request closed without merging**